Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -203,6 +203,13 @@ class KillOldProcesses(shell.Compile):
     description = ["killing old processes"]
     descriptionDone = ["killed old processes"]
     command = ["python3", "Tools/CISupport/kill-old-processes", "buildbot"]
+    
+
+class MaybePruneCoreSymbolicationdCache(shell.Compile):
+    name = "maybe-prune-coresymbolicationd-cache"
+    description = ["pruning coresymbolicationd cache if it is over 10GB in size"]
+    descriptionDone = ["pruned coresymbolicationd cache"]
+    command = ["bash", "Tools/CISupport/Shared/maybe-delete-directory.bash", "/System/Library/Caches/com.apple.coresymbolicationd"]
```

Comment:
With the shebang and executable bit set, this can be just `command = ["Tools/CISupport/Shared/maybe-delete-directory.bash", ...`.